src/hashtable.c

```diff
@@ -1824,7 +1824,7 @@ size_t hashtableScanDefrag(hashtable *ht, size_t cursor, hashtableScanFunction f
          * index pointed to by the cursor in the smaller table. */
         do {
             /* Emit entries in the larger table at this cursor, if this index
-             * hash't already been rehashed. */
+             * hasn't already been rehashed. */
```

Comment:
I think this was an in-the-neighborhood fix. (A rule against this typo could be added if people expect it to recur.)